api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -46,14 +49,40 @@ public FireCloudServiceImpl(Provider<WorkbenchConfig> configProvider,
       Provider<ProfileApi> profileApiProvider,
       Provider<BillingApi> billingApiProvider,
       Provider<GroupsApi> groupsApiProvider,
+      Provider<StatusApi> statusApiProvider,
       Provider<WorkspacesApi> workspacesApiProvider) {
     this.configProvider = configProvider;
     this.profileApiProvider = profileApiProvider;
     this.billingApiProvider = billingApiProvider;
     this.groupsApiProvider = groupsApiProvider;
+    this.statusApiProvider = statusApiProvider;
     this.workspacesApiProvider = workspacesApiProvider;
   }
 
+  @Override
+  public boolean getFirecloudStatus() {
+    StatusApi statusApi = statusApiProvider.get();
+    try {
+      statusApi.status();
+    } catch (ApiException e) {
+      String response = e.getResponseBody();
+      JSONObject errorBody = new JSONObject(response);
+      JSONObject subSystemStatus = errorBody.getJSONObject("systems");
+      if (subSystemStatus != null) {
+        return systemOkay(subSystemStatus, "Thurloe") &&
```

Comment:
Please add constants for these.

If any of these fail, would we want to shut off access to workbench entirely? Or just warn the user that things may not work right?

Reply:
I will add constants for these. 

My intent is to avoid shutting off access to workbench entirely, and just warn the user that things might not work correctly.

I still need to build out the UI for it. That's my next task, but I wanted to figure out the basics of the API first.